### PR TITLE
Make excessiveblocksize parameter and setexcessiveblock RPC command conistent

### PR DESCRIPTION
**Pull request by sickpig**

### CODE REVIEW

File: src/unlimited.cpp [hunk 1]
```diff
@@ -1091,15 +1091,15 @@ UniValue setexcessiveblock(const UniValue &params, bool fHelp)
...
+    uint64_t ebs = 0;
     if (params[0].isNum())
         ebs = params[0].get_int64();
     else
     {
         string temp = params[0].get_str();
         if (temp[0] == '-')
-            boost::throw_exception(boost::bad_lexical_cast());
-        ebs = boost::lexical_cast<unsigned int>(temp);
+            throw runtime_error("EB has to be a positive number");
```
Comment (gandrewstone) [inline src/unlimited.cpp]: EB may be our internal shorthand but we should not use it as part of the UI

Comment (sickpig) [reply src/unlimited.cpp]: ok will use "excessive block size instead".

Comment (sickpig) [reply src/unlimited.cpp]: fixed

**Comment by ptschip** [comment]
needs rebase

**Comment by sickpig** [comment]
rebased

**Pull request merged**